medmodels/medrecord/querying.pyi

```diff
@@ -175,19 +179,62 @@ class SingleValueOperand:
     ) -> None: ...
     def clone(self) -> SingleValueOperand: ...
 
+class AttributesTreeOperand:
+    def max(self) -> MultipleAttributesOperand: ...
+    def min(self) -> MultipleAttributesOperand: ...
+    def count(self) -> MultipleAttributesOperand: ...
```

Comment:
no mean, mode, median, std or var here? I can easily think of the reason but just wanted you to check it just in case

Reply:
IMO it just doesn't make sense to get the `mean`, `mode`, `median`, `std` or `var` of a node (or edge) attribute. Most of them will be strings anyway which wouldn't even support these operations